package/tensorflowmodel-mobilenet-v1-0.25-128-py/.cm/meta.json

```diff
@@ -1,45 +1,44 @@
 {
-  "check_exit_status": "yes", 
+  "check_exit_status": "yes",
   "customize": {
-    "force_ask_path": "yes", 
     "install_env": {
-      "MODEL_MOBILENET_MULTIPLIER": "0.25", 
-      "MODEL_MOBILENET_RESOLUTION": "128", 
-      "MODEL_MOBILENET_VERSION": "1", 
-      "MODEL_WEIGHTS_ARE_CHECKPOINTS": "YES", 
-      "MODULE_FILE": "tf-mobilenet-model.py", 
-      "PACKAGE_NAME": "mobilenet_v1_0.25_128_2017_06_14.tar.gz", 
-      "PACKAGE_URL": "http://download.tensorflow.org/models", 
+      "MODEL_MOBILENET_MULTIPLIER": "0.25",
+      "MODEL_MOBILENET_RESOLUTION": "128",
+      "MODEL_MOBILENET_VERSION": "1",
+      "MODEL_WEIGHTS_ARE_CHECKPOINTS": "YES",
+      "MODULE_FILE": "mobilenet-model.py",
+      "PACKAGE_NAME": "mobilenet_v1_0.25_128_2017_06_14.tar.gz",
+      "PACKAGE_URL": "http://download.tensorflow.org/models",
       "WEIGHTS_FILE": "mobilenet_v1_0.25_128.ckpt"
-    }, 
-    "no_os_in_suggested_path": "yes", 
-    "no_ver_in_suggested_path": "yes", 
-    "skip_file_check": "yes", 
+    },
+    "no_os_in_suggested_path": "yes",
+    "no_ver_in_suggested_path": "yes",
+    "skip_file_check": "yes",
     "version": "ImageNet"
```

Comment:
For the `npy` weights used by ArmCL, the version is different for each package e.g. `v1_0.25_128`.